redcap_bridge/project_building.py

```diff
@@ -212,6 +218,5 @@ def extract_customization(project_csv, export_custom_csv, *template_parts):
     if export_custom_csv is not None:
         custom_df.to_csv(export_custom_csv, index=True)
 
-
```

Comment:
removing this line is not pep8 compatible. I think it require 2 empty lines befor starting the `if __name__=='__main__':`